src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  padding: 20px;
+  background-color: black;
+}
+
+div {
+  border: 1px solid #e75551;
+}
+
+.blue-box {
+  width: 350px;
+  background-color: #308dc0;
+  padding: 10px;
+  margin-bottom: 40px;
+  box-sizing: border-box;
+}
+
+.orange-box {
+  background-color: #ffa500;
+  padding: 10px;
+}
+
+.text-box {
+  background-color: #ddd;
+  padding: 10px;
+}
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.

Reply:
Added classes instead of tag names